packages/conda-store/src/condaStoreEnvironmentManager.ts

```diff
@@ -91,12 +101,17 @@ export class CondaStoreEnvironmentManager implements IEnvironmentManager {
     this._isDisposed = true;
   }
 
+  updateSettings(settings: ISettingRegistry.ISettings): void {
+    this.baseUrl = settings.get('condaStoreUrl').composite as string;
+  }
+
   private packageManager = new CondaStorePackageManager();
   private _environmentChanged = new Signal<
     IEnvironmentManager,
     Conda.IEnvironmentChange
   >(this);
   private _isDisposed = false;
+  private baseUrl = '';
```

Comment:
Please suffix private variable with `_` - actually the `packageManager` should be updated too.

```suggestion
  private _baseUrl = '';
```
Question: as the settings registry is optional don't you want to define a default value?

Reply:
Hey, that's a good point - you don't _need_ the settings registry to run this plugin. I've added a default value here.